content/manuals/build/bake/targets.md

```diff
@@ -96,6 +96,42 @@ command.
 $ docker buildx bake all
 ```
 
+## Pattern matching for targets and groups
+
+Bake supports shell-style wildcard patterns when specifying target or grouped targets.
+This makes it easier to build multiple targets without listing each one explicitly.
+
+Supported patterns:
+
+- `*` matches any sequence of characters
+- `?` matches any single character
+- `[abc]` matches any character in brackets
+
+Examples: 
+
+```console
+# Match all targets starting with 'foo-'
+$ docker buildx bake foo-*
+
+# Match all targets
+$ docker buildx bake *
```

Comment:
We should probably call out that the value must be quoted to prevent the user's shell from expanding.  Without quotes, the shell will expand to files in the current directory, which will fail in most cases, e.g.;

```bash
docker buildx bake *
[+] Building 0.0s (1/1) FINISHED                                                                                docker:desktop-linux
 => [internal] load local bake definitions                                                                                      0.0s
 => => reading docker-bake.hcl 4.02kB / 4.02kB                                                                                  0.0s
ERROR: failed to find target api
```